be/src/exec/pipeline/pipeline_driver_dispatcher.cpp

```diff
@@ -87,8 +116,11 @@ void GlobalDriverDispatcher::run() {
             // query context has ready drivers to run, so extend its lifetime.
             query_ctx->extend_lifetime();
             auto status = driver->process(runtime_state);
-            this->_driver_queue->update_statistics(driver);
-
+            if constexpr (is_use_workgroup) {
+                this->_driver_queue->update_statistics(driver);
+            } else {
+                this->_driver_queue->get_sub_queue(queue_index)->update_accu_time(driver);
```

Comment:
We could update statistics in `QuerySharedDriverQueue::update_statistics`, and then we needn't judge whether `is_use_workgroup` here..